tests/test_config.py

```diff
@@ -174,3 +175,57 @@ def test_requierd_nothing(self):
                 settings={},
                 env={},
             )
+
+
+class TestVerbosity(unittest.TestCase):
+    """Tests for the verbosity configuration from caramel.config"""
+
```

Comment:
All multi-dimensional data types should be documented with what they represent.  ( Comments for documentation, not naked strings)

Reply:
They have doc-strings that does exactly that?

Reply:
Doc strings only exists for functions / classes.   Other strings in places are just evaluated as strings.  That was the reason for the comment. ie. complex data structures are documented with a comment above them. sorry if I was unclear with that explanation